src/civis/ml/_model.py

```diff
@@ -555,6 +559,8 @@ def __setstate__(self, state):
         self._condition = threading.Condition()
         self.client = APIClient()
         self.poller = self.client.scripts.get_containers_runs
+        self._next_polling_interval = 1
+        self._use_geometric_polling = True
```

Comment:
`ModelFuture` objects are designed to be pickle-able, so we need to restore the default values of these polling-related attributes upon un-pickling.